Week01/Problem01/bpeng_01.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+"""
+Spyder Editor
+
+This is a temporary script file.
+"""
+class Solution:
+    def sumofmultiples(self,limit):
+        numbers = []
+        for i in range(1,limit):
+            if i % 3 == 0 or i % 5 == 0:
+                numbers.append(i)
```

Comment:
Be careful, if 'limit' becomes large this may use a significant amount of memory. Try simply updating a single variable.